lib/internal/fs/watchers.js

```diff
@@ -309,11 +316,15 @@ async function* watch(filename, options = kEmptyObject) {
     persistent = true,
     recursive = false,
     encoding = 'utf8',
+    maxQueue = 2048,
```

Comment:
Can you add some documentation of this to `doc/api/fs.md`?

Reply:
Done in 6ae7e7d0a8be3a5bce0e13f55a4e24766bcb8d84